### DISCUSSION TITLE
Change Android OS language within yaml flow

### DISCUSSION THREAD

**Discussion by MichaelLutzDL** [mobile-dev-inc/Maestro#881] [Q&A] [answered]
Hi,

I would like to write a flow to change the OS system language on Android 13. Unfortunately I have not found a way to drag and release languages. Is there a possibility to do so or an alternative which could be suitable?

Best Michael

**Comment by axelniklasson** [comment, marked as answer]
Hi there @MichaelLutzDL, thanks for the suggestion! Changing the Android OS language is not something we plan on tackling, but there is a feature request open for launching the app with a specific locale that we want to prioritize at some point. You can follow along more here: https://github.com/mobile-dev-inc/maestro/issues/393. Thanks!
